Converters/Navisworks/Speckle.Converters.NavisworksShared/DataHandlers/BasePropertyHandler.cs

```diff
@@ -17,17 +17,35 @@ ModelPropertiesExtractor modelPropertiesExtractor
 
     if (propertySets != null)
     {
-      foreach (var category in propertySets.Where(c => c.Key != "Internal"))
+      foreach (var category in propertySets.Where(c => c.Key != "Internal" && c.Key != "Transform"))
       {
+        var excludedProperties = new List<string> { "Hidden", "Required", "Internal_Type" };
```

Comment:
I would put this into outside of for each loop since we do not populate it, there is no need to generate the list every time

Reply:
Changed to a read-only class property